packages/context/src/interceptor.ts

```diff
@@ -315,3 +315,36 @@ function invokeInterceptors(
   };
   return next();
 }
+
+/**
+ * A proxy handler that applies interceptors
+ *
+ * See https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Proxy
+ */
+export class InterceptorHandler<T extends object> implements ProxyHandler<T> {
+  constructor(private context = new Context()) {}
```

Comment:
Under which circumstances would it make sense to use this handler with an empty context? Can we remove the default argument value?

Reply:
If the method does not have `interceptors` from binding keys, `context` is not required.